src/server/scripts/EasternKingdoms/ScarletEnclave/chapter2.cpp

```diff
@@ -214,35 +212,36 @@ class npc_koltira_deathweaver : public CreatureScript
 public:
     npc_koltira_deathweaver() : CreatureScript("npc_koltira_deathweaver") { }
 
-    bool OnQuestAccept(Player* player, Creature* creature, const Quest* quest) override
-    {
-        if (quest->GetQuestId() == QUEST_BREAKOUT)
-        {
-            creature->SetStandState(UNIT_STAND_STATE_STAND);
-            creature->setActive(true);
-
-            if (npc_escortAI* pEscortAI = CAST_AI(npc_koltira_deathweaver::npc_koltira_deathweaverAI, creature->AI()))
-                pEscortAI->Start(false, false, player->GetGUID());
-        }
-        return true;
-    }
-
     CreatureAI* GetAI(Creature* creature) const override
     {
         return new npc_koltira_deathweaverAI(creature);
     }
 
     struct npc_koltira_deathweaverAI : public npc_escortAI
     {
-        npc_koltira_deathweaverAI(Creature* creature) : npc_escortAI(creature), summons(me)
-        {
-            me->SetReactState(REACT_DEFENSIVE);
-        }
-
         uint32 m_uiWave;
         uint32 m_uiWave_Timer;
         ObjectGuid m_uiValrothGUID;
         SummonList summons;
+        bool active;
+
+        npc_koltira_deathweaverAI(Creature* creature) : npc_escortAI(creature), summons(me)
```

Comment:
It is a waste of time, because what we have to do is rewrite the entire script. It was a temporary solution, to prevent the error from continuing. But it is not definitive. I should have closed this pull request, but I forgot to do it. This solution is attacked with wires.